src/plugins/discover/public/application/components/sidebar/discover_field_search.tsx

```diff
@@ -75,13 +77,19 @@ export interface Props {
    * types for the type filter
    */
   types: string[];
+  isEnhancementsEnabledOverride: boolean;
```

Comment:
Nit: would be nice to have a comment to describe  `isEnhancementsEnabledOverride`